core/commands/add.go

```diff
@@ -30,9 +30,11 @@ const (
 
 var AddCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Add a file or directory to ipfs.",
+		Tagline: "Add a file to ipfs.",
```

Comment:
Because with `-r` or `-w` you're adding more than a file. Seemed to me like that would be pretty common, and that it should be clear that this command also works for dirs.